llvm/test/CodeGen/RISCV/zbb-logic-neg-imm.ll

```diff
@@ -273,37 +240,21 @@ define i32 @compl(i32 %x) {
 }
 
 define i32 @orlow12(i32 %x) {
```

Comment:
With Zbs this is:

        ori     a0, a0, 2047
        bseti   a0, a0, 11

and is not affected by this change. Shall I add `+zbs` or test both with and without?

Reply:
I would add RUN lines with `+zbs`, rather than adding `+zbs` to the existing RUN lines.

Reply:
Done